statsmodels/stats/power.py

```diff
@@ -172,39 +172,48 @@ def normal_sample_size_one_tail(diff, power, alpha, std_null=1.,
     """explicit sample size computation if only one tail is relevant
 
     The sample size is based on the power in one tail assuming that the
-    alternative is in the tail where the test has power that increases
-    with sample size.
-    Use alpha/2 to compute the one tail approximation to the two-sided
-    test, i.e. consider only one tail of two-sided test.
+    alternative is in the tail where the test has power that increases with
+    sample size. Use alpha/2 to compute the one tail approximation to the
+    two-sided test, i.e. consider only one tail of two-sided test.
 
     Parameters
     ----------
     diff : float
         difference in the estimated means or statistics under the alternative.
-    power : float in interval (0,1)
-        power of the test, e.g. 0.8, is one minus the probability of a type II error.
-        Power is the probability that the test correctly rejects the Null
-        Hypothesis if the Alternative Hypothesis is true.
+    power : float in interval [alpha,1)
+        power of the test, e.g. 0.8, is one minus the probability of a type II
+        error. Power is the probability that the test correctly rejects the
+        Null Hypothesis if the Alternative Hypothesis is true.
     alpha : float in interval (0,1)
-        significance level, e.g. 0.05, is the probability of a type I
-        error, that is wrong rejections if the Null Hypothesis is true.
+        significance level, e.g. 0.05, is the probability of a type I error,
+        that is wrong rejections if the Null Hypothesis is true.
     std_null : float
         standard deviation under the Null hypothesis without division by
         sqrt(nobs)
     std_alternative : float
-        standard deviation under the Alternative hypothesis without division
-        by sqrt(nobs)
+        standard deviation under the Alternative hypothesis without division by
+        sqrt(nobs)
 
     Returns
     -------
     nobs : float
         sample size to achieve the desired power
 
     """
+    if std_alternative is None:
+        std_alternative = 0
```

Comment:
this should be = std_null

Reply:
I'll make this change and note it in the documentation, but I don't think it's appropriate for proportions due to [my last comment](https://github.com/statsmodels/statsmodels/pull/8507#issuecomment-1306322495).

Reply:
Changed in [`b6d264d7`](https://github.com/statsmodels/statsmodels/pull/8507/commits/b6d264d74a29a3883c8d7eef2ec56118d2970564), and added documentation to the `std_alternative` parameter noting this choice of default.